cmd/vic-machine/create/create.go

```diff
@@ -420,6 +429,16 @@ func (c *Create) ProcessBridgeNetwork() error {
 	if err != nil {
 		return cli.NewExitError(fmt.Sprintf("Error parsing bridge network ip range: %s. Range must be in CIDR format, e.g., 172.16.0.0/12", err), 1)
 	}
+	width, err := strconv.Atoi(c.BridgeNetworkWidth)
```

Comment:
Better to move validation codes into validator.network() api, so rest apis will share the same validation codes.

Reply:
I leave the validation for rest api defined in swagger using minimum and maximum.